crypto/mime.go

```diff
@@ -46,21 +51,69 @@ func (keyRing *KeyRing) DecryptMIMEMessage(
 	callbacks.OnEncryptedHeaders("")
 }
 
+type MIMEMessage struct {
+	Headers        []string
```

Comment:
@wussler What type would make the most sense for Headers in your view ?
I think we can leave the behavior of not parsing the headers for now, but we should at least get the type right to avoid api changes.

Reply:
Probably a string map?